simq/README.md

```diff
@@ -0,0 +1,120 @@
+# SimQ - Face Quality Assessment
+
+SimQ is an Android library for assessing the quality of face images. It analyzes multiple quality metrics to provide a comprehensive quality score between 0.0 and 1.0.
+
+## Features
+
+SimQ evaluates face images based on four key metrics:
+
+- **Alignment**: Evaluates face pose angles (pitch, yaw, roll) and eye state
+- **Blur**: Measures image sharpness using Laplacian variance
+- **Brightness**: Assesses image luminance levels
+- **Contrast**: Evaluates pixel intensity variation
+
+## Installation
+
+Add the dependency to your `build.gradle.kts`:
+
+```kotlin
+implementation("com.simprints.biometrics:simq:2025.4.0")
+```
+
+## Basic Usage
+
+```kotlin
+// Initialize SimQ with default parameters
+val simQ = SimQ()
+
+// Calculate quality score for a face image
+val qualityScore = simQ.calculateFaceQuality(
+    bitmap = faceBitmap,
+    pitch = 5.0,
+    yaw = -3.0,
+    roll = 0.0
+)
+
+// Quality score ranges from 0.0 (poor) to 1.0 (excellent)
+if (qualityScore >= 0.6) {
+    // Image quality is sufficient
+}
+```
+
+## Advanced Usage
+
+### Custom Weights
+
+Customize how much each metric contributes to the final score:
+
+```kotlin
+val customWeights = QualityWeights(
+    alignment = 0.25,
+    blur = 0.35,
+    brightness = 0.25,
+    contrast = 0.10,
+    eyeOpenness = 0.05
+)
+
+val simQ = SimQ(faceWeights = customWeights)
+```
+
+### Custom Parameters
+
+Adjust the thresholds for each quality metric:
+
+```kotlin
+val customParameters = QualityParameters(
+    maxAlignmentAngle = 15.0,
+    minBlur = 60_000.0,
+    maxBlur = 120_000.0,
+    minBrightness = 40.0,
+    optimalBrightnessLow = 90.0,
+    optimalBrightnessHigh = 140.0,
+    maxBrightness = 180.0,
+    minContrast = 35.0,
+    maxContrast = 50.0
+)
+
+val simQ = SimQ(faceParameters = customParameters)
+```
+
+## Parameters Reference
+
+### calculateFaceQuality()
```

Comment:
Fine for now, but I would suggest adding Dokka and generating documentation based on code comments, as this will get outdated sooner or later.